hazelcast/src/main/java/com/hazelcast/client/impl/protocol/template/EventResponseTemplate.java

```diff
@@ -175,16 +161,45 @@ void Entry(@Nullable Data key, @Nullable Data value, @Nullable Data oldValue, @N
     @EventResponse(EventMessageConst.EVENT_CACHEPARTITIONLOST)
     void CachePartitionLost(int partitionId, String uuid);
 
+    /**
+     * @param name          Name of the cache.
+     * @param key           The key for the entry.
+     * @param sourceUuid    source uuid
+     * @param partitionUuid partition uuid
+     * @param sequence      sequence number of invalidation event
+     */
+    @EventResponse(EventMessageConst.EVENT_CACHEINVALIDATION)
+    void CacheInvalidation(String name, @Nullable Data key, @Nullable String sourceUuid,
```

Comment:
sourceUuid is of type string and partitionUuid is now of different type UUID. This will be confusing for people.